### DISCUSSION TITLE
Wallpaper scaled option + background color

### DISCUSSION THREAD

**Discussion by Thorinori** [regolith-linux/regolith-desktop#883] [Q&A] [answered]
Hello, I currently have a wallpaper set via Xresources using the regolith.wallpaper.file key and have it set to scaled via regolith.wallpaper.options, and I can't for the life of me figure out if I can change the default background color that is applied to either side of the scaled image.

![image](https://github.com/regolith-linux/regolith-desktop/assets/8883561/cad19976-7914-44ce-ae2e-bc30f8b70fef)
More specifically, I want to set the blue to more of a black baseline so it looks more like a smooth transition with the wallpaper itself, but I can't seem to find an Xresources key that does that. Any help would be appreciated! As for other things I thought might be related, I am using blackhole for my regolith look and Ayu-Dark for my gtk theme (I am thinking it could maybe be this one, and if that is the case, any suggestions for a dark dark black gtk theme would be appreciated too)

**Comment by Thorinori** [comment]
I realized I was overthinking it, and that it was likely due to the transparency on the image. I converted to jpeg and it seems to be working properly now. Would still like to know how to change the base color if possible though for in the future if I want to retain transparency!

**Reply by kgilmer** [reply, marked as answer]
Hi @Thorinori , in the X11 session, the wallpaper is controlled by `gnome-flashback` via configuration in gsettings.  I haven't tested it but believe you can adjust the wallpaper color with the following key:

```
$ gsettings list-keys org.gnome.desktop.background
color-shading-type
picture-opacity
picture-options
picture-uri
picture-uri-dark
primary-color
secondary-color
show-desktop-icons
$ gsettings describe org.gnome.desktop.background primary-color
Left or Top color when drawing gradients, or the solid color.
$ gsettings set org.gnome.desktop.background primary-color #somecolor
```

**Reply by Thorinori** [reply]
Ah alright, that would make sense to me. Thanks for the info!
